packages/shortest/src/utils/errors.ts

```diff
@@ -1,43 +1,67 @@
-import { ZodError } from "zod";
+import { z, ZodError } from "zod";
 
-// eslint-disable-next-line zod/require-zod-schema-types
-export type ConfigErrorType =
-  | "duplicate-config"
-  | "file-not-found"
-  | "invalid-config"
-  | "no-config";
-export class ConfigError extends Error {
+export class ShortestError extends Error {
```

Comment:
Adding our own error class so that we can filters the errors that need to be caught. Now we do a catch-all, which is not great.